sv/models/evenements.py

```diff
@@ -59,12 +60,27 @@ class Meta:
         verbose_name = "Évènement"
         verbose_name_plural = "Évènements"
         constraints = []
+        unique_together = ("numero_annee", "numero_evenement")
```

Comment:
Je propose la nouvelle notation car le unique_together va être déprécié (https://docs.djangoproject.com/en/5.1/ref/models/options/#django.db.models.Options.unique_together)
```suggestion
constraints = [
        models.UniqueConstraint(
            fields=['numero_annee', 'numero_evenement'], 
            name='unique_evenement_numero'
        )
    ]
```

Reply:
Je ne vois pas dans cette documentation que unique_together va être déprécié. Je vais réaliser le changement pour être plus pérenne, il faudrait reproduire ce changement sur toutes les conditions / checks

Reply:
Dans le cadre vert ;) -> https://docs.djangoproject.com/en/5.1/ref/models/options/#django.db.models.Options.unique_together

> unique_together may be deprecated in the future.

Reply:
exact ! mais ça sera fait si toutefois c'est le cas :)